.github/ISSUE_TEMPLATE.md

```diff
@@ -13,11 +13,13 @@
 1. [step 1]
 2. [step 2]
 
-**Dump file**
-[If you have a dump file: zip it before you drag & drop it here.]
+**Dump file:**
+[In case the game asked you to provide one after the game crashed: You can find it at `%userprofile%\documents\OpenRCT2` Look for the latest .DMP file, zip it and drag & drop it here.]
```

Comment:
Does someone know if `          ` works for the path to the location of the dump file as I can't test it now, as these changes aren't merged yet?